### ISSUE TITLE
Binance No market symbol VTHO/BTC or ONG/BTC

### ISSUE THREAD

**Issue by baudbox** [freqtrade/freqtrade#1272]
Dear Team,

Telegram rpc /balance cant' be retrieved and I've got htis following message:

  ```
2018-10-10 07:43:51,153 - freqtrade.exchange - WARNING - get_ticker() returned exception: "Could not load ticker due to ExchangeError. Message: binance No market symbol VTHO/BTC"
  ```

This issue has been already raised in August in #1111 but reappears in my case. The commit #1135 fixed it for a while. 

I'm on tha latest commit from develop branch.

  * Python Version: Python 3.6.3
  * CCXT version: ccxt==1.17.375
  * Branch: Develop
  * Last Commit ID: eee0958a588acad0bcfa8c34bbf1484bda917260
  
Bot was restarted, both altcoin are not defined in whitelist at all, neither appear into the DB

Thanks for any kind of help

### Relevant code exceptions or logs:

  ```
2018-10-10 07:43:48,150 - freqtrade.exchange - INFO - returning cached ticker-data for NCASH/BTC
2018-10-10 07:43:48,151 - freqtrade.exchange - INFO - returning cached ticker-data for ONT/BTC
2018-10-10 07:43:49,153 - freqtrade.exchange - INFO - returning cached ticker-data for WAN/BTC
2018-10-10 07:43:51,153 - freqtrade.exchange - WARNING - get_ticker() returned exception: "Could not load ticker due to ExchangeError. Message: binance No market symbol VTHO/BTC"
2018-10-10 07:43:51,153 - freqtrade.exchange - WARNING - retrying get_ticker() still for 3 times
2018-10-10 07:43:51,155 - freqtrade.exchange - WARNING - get_ticker() returned exception: "Could not load ticker due to ExchangeError. Message: binance No market symbol VTHO/BTC"
2018-10-10 07:43:51,156 - freqtrade.exchange - WARNING - retrying get_ticker() still for 2 times
2018-10-10 07:43:51,157 - freqtrade.exchange - WARNING - get_ticker() returned exception: "Could not load ticker due to ExchangeError. Message: binance No market symbol VTHO/BTC"
2018-10-10 07:43:51,158 - freqtrade.exchange - WARNING - retrying get_ticker() still for 1 times
2018-10-10 07:43:51,159 - freqtrade.exchange - WARNING - get_ticker() returned exception: "Could not load ticker due to ExchangeError. Message: binance No market symbol VTHO/BTC"
2018-10-10 07:43:51,160 - freqtrade.exchange - WARNING - retrying get_ticker() still for 0 times
2018-10-10 07:43:51,162 - freqtrade.exchange - WARNING - get_ticker() returned exception: "Could not load ticker due to ExchangeError. Message: binance No market symbol VTHO/BTC"
2018-10-10 07:43:51,163 - freqtrade.exchange - WARNING - Giving up retrying: get_ticker()
2018-10-10 07:43:51,164 - freqtrade.exchange - WARNING - get_ticker() returned exception: "Could not load ticker due to ExchangeError. Message: binance No market symbol ONG/BTC"
2018-10-10 07:43:51,165 - freqtrade.exchange - WARNING - retrying get_ticker() still for 3 times
2018-10-10 07:43:51,167 - freqtrade.exchange - WARNING - get_ticker() returned exception: "Could not load ticker due to ExchangeError. Message: binance No market symbol ONG/BTC"
2018-10-10 07:43:51,167 - freqtrade.exchange - WARNING - retrying get_ticker() still for 2 times
2018-10-10 07:43:51,169 - freqtrade.exchange - WARNING - get_ticker() returned exception: "Could not load ticker due to ExchangeError. Message: binance No market symbol ONG/BTC"
2018-10-10 07:43:51,169 - freqtrade.exchange - WARNING - retrying get_ticker() still for 1 times
2018-10-10 07:43:51,171 - freqtrade.exchange - WARNING - get_ticker() returned exception: "Could not load ticker due to ExchangeError. Message: binance No market symbol ONG/BTC"
2018-10-10 07:43:51,172 - freqtrade.exchange - WARNING - retrying get_ticker() still for 0 times
2018-10-10 07:43:51,173 - freqtrade.exchange - WARNING - get_ticker() returned exception: "Could not load ticker due to ExchangeError. Message: binance No market symbol ONG/BTC"
2018-10-10 07:43:51,174 - freqtrade.exchange - WARNING - Giving up retrying: get_ticker()
2018-10-10 07:43:51,622 - freqtrade.rpc.telegram - WARNING - Telegram NetworkError: Message is too long! Trying one more time.
2018-10-10 07:43:51,673 - freqtrade.rpc.telegram - WARNING - TelegramError: Message is too long! Giving up on that message.
2018-10-10 07:44:03,940 - freqtrade.freqtradebot - INFO - checking sell
2018-10-10 07:44:03,942 - freqtrade.freqtradebot - INFO - Found no sell signals for whitelisted currencies. Trying again..
  ```

**Comment by xmatthias** [comment]
you're right, i've now found a scenario on how to reliably reproduce the issue (using a database with open pairs not available on the exchange ...) 
hacky - but works (rather fails) reliably.

Thanks for reporting.

**Comment by xmatthias** [comment]
fixed in #1276
